bundle-generation/generate-bundles.py

```diff
@@ -498,13 +538,23 @@ def updateDeployments(helmChart, exclusions):
 
         pod_template = deploy['spec']['template']
         pod_template['metadata']['labels']['ocm-antiaffinity-selector'] = deploy['metadata']['name']
+        if sizes:
+            for  sizDeployment in sizes["deployments"]:
+                if sizDeployment["name"] == deploy["metadata"]["name"]:
+                    for i in deploy['spec']['template']['spec']['containers']:
+                        for sizContainer in sizDeployment["containers"]:
+                            if sizContainer["name"] == i["name"]:
+                                i['resources'] = 'REPLACE-' + i['name']
 
         pod_template_spec = pod_template['spec']
         pod_template_spec['affinity'] = deploySpec['affinity']
         pod_template_spec['tolerations'] = ''
         pod_template_spec['hostNetwork'] = False
         pod_template_spec['hostPID'] = False
         pod_template_spec['hostIPC'] = False
+        
+        if 'automountServiceAccountToken' not in pod_template_spec:
+            pod_template_spec['automountServiceAccountToken'] = False
```

Comment:
I believe we can comment this out for now, earlier when it was committed it broke the build for several components.

We can re-enable it in another PR; however, for this one we should not include it in the sizes change.